plugins/draw-tools.js

```diff
@@ -383,12 +393,15 @@ window.plugin.drawTools.optCopy = function() {
 }
 
 window.plugin.drawTools.optExport = function() {
-  var data = localStorage[window.plugin.drawTools.KEY_STORAGE];
-  window.saveFile(data, 'IITC-drawn-items.json', 'application/json');
+  if (window.localStorage[window.plugin.drawTools.KEY_STORAGE] === '' ||
+      window.localStorage[window.plugin.drawTools.KEY_STORAGE] === undefined)
```

Comment:
Again, all this mess is hard to read.
Why not write like this:
```js
  var drawTools = window.plugin.drawTools;
  if (localStorage[drawTools.KEY_STORAGE] === '' ||
      localStorage[drawTools.KEY_STORAGE] === undefined)
```

Or better this:
```js
  var drawTools = window.plugin.drawTools;
  if (!localStorage[drawTools.KEY_STORAGE])
```

Reply:
On the one hand you ask me not to change lines that do not belong to the commit, OTOH you ask me to change the way the code was written until now.

I try to follow they way the plugin was written until now. If we want to remove "window." and "window.plugin" we have to do that in one single commit, but not every now and then.

As KEY_STORAGE can be set to '' (empty string), the short variant is insufficient.

Reply:
> On the one hand you ask me not to change lines that do not belong to the commit,

Right. As I specified earlier - those concerns are about original code.
Just to make sure that we both (and all others) are sharing similar view.

As for changing this bad code: definitely there is no sense to change just 3 lines leaving the rest kilobytes in old state.
So yes, if you make some minor change - better not mix it with refactoring.
Still, this legacy requires refactoring. Just keep it separately (separate commits, or separate PR).

Reply:
> As KEY_STORAGE can be set to '' (empty string), the short variant is insufficient.

Why should we allow empty string?

Reply:
After checking the old code. These two lines are legacy code. I only added a linebreak for readability.
The newline itself was added in an earlier PR (likely MPE).